packages/test/test-utils/src/testFluidObject.ts

```diff
@@ -135,7 +135,6 @@ export class TestFluidObjectFactory implements IFluidDataStoreFactory {
 
     public async instantiateDataStore(
         context: IFluidDataStoreContext,
-        existing: boolean,
```

Comment:
I think this should stay, actually. It also won't compile, I think this parameter is used below.